pkg/common/kubernetes/informers.go

```diff
@@ -104,3 +106,9 @@ func (im *InformerManager) AddNodeListener(add, remove func(obj interface{}), up
 func (im *InformerManager) Listen() {
 	go im.informerFactory.Start(im.stopCh)
 }
+
+// GetNodeList gets node list
+func (im *InformerManager) GetNodeList() ([]*v1.Node, error) {
+	nodeLister := im.informerFactory.Core().V1().Nodes().Lister()
```

Comment:
Is it possible to reuse the node list capture below instead of creating a new informer with a separate list? or is it a chicken/egg initialization thing?
https://github.com/kubernetes/cloud-provider-vsphere/blob/master/pkg/cloudprovider/vsphere/nodemanager.go#L56

Reply:
I think `nodeRegUUIDMap` is still empty when initializing route provider. If it is not good to add a func in InformerManager, I can move it into RouteProvider `Initialize`